floor/floor/processor/utils_test.py

```diff
@@ -1,72 +1,79 @@
 import datetime
 import mock
-from freezegun import freeze_time
 from unittest import TestCase
 from floor.processor.utils import clocked
+from floor.processor.base import RenderContext
 
 
-class FakeProcessor:
-    bpm = 120
-
 
 class UtilsTestCase(TestCase):
     def test_clocked(self):
         wrapped_fn = mock.Mock()
         wrapped_fn.side_effect = ['a', 'b', 'c', 'd']
         fn = clocked()(wrapped_fn)
 
-        with freeze_time() as frozen_datetime:
```

Comment:
Deltas in this file showcase one of the benefits of a global rendering clock: we no longer need to resort to patching/mocking `time.time()` to get a processor to generate a frame at a specific point in time.